azuredevops/internal/acceptancetests/resource_branchpolicy_test.go

```diff
@@ -23,7 +23,7 @@ func TestAccBranchPolicyMinReviewers_CreateAndUpdate(t *testing.T) {
 		Providers: testutils.GetProviders(),
 		Steps: []resource.TestStep{
 			{
-				Config: getMinReviewersHcl(true, true, 1, false),
+				Config: getMinReviewersHcl(true, true, 1, false, "null", "exact"),
```

Comment:
`repoRef` should be set when matchType=Exact
```suggestion
				Config: getMinReviewersHcl(true, true, 1, false, "\"refs/heads/release\"", "Exact"),
```